packages/pds/src/sequencer/events.ts

```diff
@@ -13,41 +14,23 @@ export const formatSeqCommit = async (
   blocksToSend.addMap(commitData.newBlocks)
   blocksToSend.addMap(commitData.relevantBlocks)
 
-  let evt: CommitEvt
-
-  // If event is too big (max 200 ops or 1MB of data)
-  if (commitData.ops.length > 200 || blocksToSend.byteSize > 1000000) {
-    const justRoot = new BlockMap()
-    const rootBlock = blocksToSend.get(commitData.cid)
-    if (rootBlock) {
-      justRoot.set(commitData.cid, rootBlock)
-    }
+  // If event is too big (max 200 ops or 2MB of data)
+  if (blocksToSend.byteSize > 2000000) {
+    throw new InvalidRequestError('Too many writes. Max event size: 2MB')
+  }
```

Comment:
The write could have already been made to the repo in this case, I believe, so this is sort of like an assertion to ensure we don't send bad data over the firehose.  I think it would be fine if this surfaced as a 500 to be honest, as this shouldn't ever happen.

Did we ever end-up enforcing a 1mb max on the record blocks within the write handlers?  I think ideally there error would come from around there, during the repo write.

Reply:
Ah good call, I like just moving this all to the write handler